mock/py/mockbuild/config.py

```diff
@@ -311,6 +312,12 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir):
     return config_opts
 
 
+def multiply_platform_multiplier(config_opts):
+    if config_opts['forcearch']:
+        if (not config_opts['macros']['%_platform_multiplier']) or \
+           (str(config_opts['macros']['%_platform_multiplier']) == '1'):
+            config_opts['macros']['%_platform_multiplier'] = 10
```

Comment:
Can this be just?
`config_opts['macros']['%_platform_multiplier'] = 10 if config_opts['forcearch'] else 1`

Reply:
Not if we want to allow admin to override (fine tune) it in config. Consider having `config_opts['macros']['%_platform_multiplier'] = 5` in config.

Reply:
Ok, I'd suggest something like this (I'm reading over again, and it looks
too magic to me):
```python
def multiply_platform_multiplier(config_opts):
    if not config_opts['forcearch']:
        # forcearch disabled, no need to increase the multiplier
        return

    if config_opts['macros']['%_platform_multiplier'] is not None:
        # multiplier set by end-user explicitly
        return

    # increase to 10 for forcearch
    config_opts['macros']['%_platform_multiplier'] = 10
```

And since I believe we can call this only once, I don't think we necessarily have to have this as
a separate method from the `check_arch_combination` (but this is not a strong opinion).

Reply:
No, wrong.  This is better:
```python
if config_opts['macros']['%_platform_multiplier'] is not None:
    # multiplier set by end-user explicitly
    return
config_opts['macros']['%_platform_multiplier'] = 10 if config_opts['forcearch'] else 1
```

It is OK because we don't have to call this multiple times.

Reply:
The default is 1 (not None).

Reply:
The logic is is really complicated;  e.g. if user decides to set the value explicitly to `1`, we reset it to `10`.

Reply:
> I don't think we necessarily have to have this as a separate method from the check_arch_combination

check_arch_combination is already quite long. I want to have it separate.

Reply:
I updated the PR and added comments to this section. Is it better or you still want to rephrase it?